rootfs/bin/test

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python /app/manage.py test ingress
```

Comment:
I think this should be removed in favour of #1049. Is this something required for this PR or would you agree?